### PR TITLE
added info about blocked state request

### PR DESCRIPTION
**Pull request by nithyaasworld**
Added blocked state example for a webhook request. 

This PR references the content added in [this PR](https://github.com/buildkite/docs/pull/1345) 

In the PR:
image

### CODE REVIEW

**Comment by nithyaasworld** [comment]
Spec failed because this PR references to a link that becomes available only after merging the PR https://github.com/buildkite/docs/pull/1345.

File: pages/apis/webhooks/build_events.md.erb [hunk 1]
```diff
@@ -39,3 +39,30 @@ Example request body:
   }
 }
 ```
+## To determine the blocked state of a build:
+
+To determine the blocked state of a build, look for ```blocked: true``` in the ```build.finished``` event
```
Comment (plaindocs) [inline pages/apis/webhooks/build_events.md.erb:44]:
```suggestion
To if a build is blocked, look for `blocked: true` in the `build.finished` event
```

File: pages/apis/webhooks/build_events.md.erb [hunk 2]
```diff
@@ -39,3 +39,30 @@ Example request body:
   }
 }
 ```
+## To determine the blocked state of a build:
```
Comment (plaindocs) [inline pages/apis/webhooks/build_events.md.erb:42]:
```suggestion
## Finding out if a build is blocked
```

File: pages/apis/webhooks/build_events.md.erb [hunk 3]
```diff
@@ -39,3 +39,30 @@ Example request body:
...
+  },
+  "sender": {
+    "id": "0adfbc27-5f72-4a91-bf61-5693da0dd9c5",
+    "name": "Some person"
+  }
+}
+```
+<div class="Docs__note">
+  <h3 class="Docs__note__heading">To determine blocked state in an eventbridge notification</h3>
+  <p>However, to determine the blocked state of a build in an eventbridge notification, look for <code>"state": "blocked". </code> <a href="/docs/integrations/amazon-eventbridge#events-build-blocked">Here</a> is a sample request for eventbridge notification.</p>
```
Comment (plaindocs) [inline pages/apis/webhooks/build_events.md.erb:67]:
```suggestion
  <h3 class="Docs__note__heading">To determine if an Eventbridge notification is blocked</h3>
  <p>However, to determine if an Eventbridge notification is blocked, look for <code>"state": "blocked". </code>, like in this <a href="/docs/integrations/amazon-eventbridge#events-build-blocked">sample Eventbridge request</a>.</p>
```

**Review by plaindocs — APPROVED** [review]
Nice clarification. :-)

**Comment by plaindocs** [comment]
@nithyaasworld if the you're adding related content (like here and #1345 you're welcome to put them in the same PR, especially if it makes the build easier. Alternatively, branch the second PR off the first one, so the build will pass).

**Pull request merged**